### PR TITLE
Update config files

### PR DESCRIPTION
**Pull request by Justinezgh**
This PR aims to change the config files to train the compressor. Here are the changes:
- for all configs: score loss -> GNLL
- change model with model_nopatch for the training without pre-training and fine-tuning (`efficientnet_v2_s_logp.yaml`)
- for all configs: "infinite" training procedure with the best model being saved. To track the best model, we use the validation mse and evaluate it every 500 steps.
- for all configs: we use a summary dim of 8.

### CODE REVIEW

**Review by b-remy — CHANGES_REQUESTED** [review]
For finetuning, i think we want to store the model which minimizes the negative log likelihood, not only the MSE

File: configs/finetune_from_pretrain_nopatch_logp.yaml [hunk 1]
```diff
@@ -38,8 +39,8 @@ trainer:
     - class_path: ModelCheckpoint
       init_args:
         dirpath: "checkpoints/"
-        monitor: "val_log_prob"
-        mode: "max"
+        monitor: "val_mse"
+        mode: "min"
```
[the remark this answers is not in the document]
Comment (b-remy) [reply configs/finetune_from_pretrain_nopatch_logp.yaml]: This is because there is no `val_log_prob` yet. We need to add this

**Comment by b-remy** [comment]
Yes, we need to add `val_log_prob` to the models.
